src/Estimators/tests/RobotDynamicsEstimator/SubModelCreatorTest.cpp

```diff
@@ -73,25 +75,9 @@ TEST_CASE("SubModel Creation")
     auto kinDyn = std::make_shared<iDynTree::KinDynComputations>();
     REQUIRE(kinDyn->loadRobotModel(mdlLdr.model()));
 
-    // Case with FT, the model will be splitted
-    RDE::SubModelCreator subModelCreatorWithFT;
-    subModelCreatorWithFT.setModelAndSensors(mdlLdr.model(), mdlLdr.sensors());
-    REQUIRE(subModelCreatorWithFT.setKinDyn(kinDyn));
-    REQUIRE(subModelCreatorWithFT.createSubModels(originalHandler));
-
-    REQUIRE(subModelCreatorWithFT.getNrOfSubModels() == (1 + mdlLdr.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE)));
-
-    // Case without FT, the model is not splitted and the only sub-model is the full model
-    RDE::SubModelCreator subModelCreatorWithoutFT;
-    subModelCreatorWithoutFT.setModelAndSensors(mdlLdr.model(), mdlLdr.sensors());
-    REQUIRE(subModelCreatorWithoutFT.setKinDyn(kinDyn));
-    auto groupFT = parameterHandler->getGroup("FT").lock()->clone();
-    groupFT->clear();
-    std::vector<std::string> emptyVector;
-    groupFT->setParameter("names", emptyVector);
-    groupFT->setParameter("frames", emptyVector);
-    parameterHandler->setGroup("FT", groupFT);
-    REQUIRE(subModelCreatorWithoutFT.createSubModels(parameterHandler));
-
-    REQUIRE(subModelCreatorWithoutFT.getNrOfSubModels() == 1);
```

Comment:
Was this part of the test not needed or ill-posed?

Reply:
That was an effect of the rebase. I fixed it.